### PR TITLE
IoC container name corrections

### PR DESCRIPTION
**Pull request by nblumhardt**
Autofac is one word (lowercase 'f'). Windsor is the IoC container from the Castle project (not CastleWindsor). Cheers!

### CODE REVIEW

File: src/docs/Dependency injection.md [hunk 1]
```diff
@@ -75,10 +75,10 @@ guideline.
 
 Currently the following Akka.NET Dependency Injection plugins are available:
 
-## AutoFac
+## Autofac
 
 In order to use this plugin, install the Nuget package with
-`Install-Package Akka.DI.AutoFac`, then follow the instructions:
+`Install-Package Akka.DI.Autofac`, then follow the instructions:
```
Comment (sean-gilliam) [inline src/docs/Dependency injection.md]: While the name is indeed Autofac, our nuget package is still Akka.DI.AutoFac.
If you wish to correct the package name, here is the repo for it
https://github.com/akkadotnet/Akka.DI.AutoFac . Until that is resolved, I suggest we keep the install instructions like they are for now.

**Comment by sean-gilliam** [comment]
LGTM except for my note.

**Comment by nblumhardt** [comment]
Thanks @sean-gilliam. Package names are case-insensitive, so `install-package akka.di.autofac` would also work. I believe even when the package name is corrected, various parts of NuGet will still show the old name (it's a bit of a mess), but I'll head over and submit a PR on the package also 👍

**Comment by nblumhardt** [comment]
Just checked the other repo, sadly the namespace/type names are also incorrectly cased, which would mean breaking changes for a fix over there. I've reverted the version in the `install-package` version instead. Cheers!

**Comment by sean-gilliam** [comment]
Thank you for contributing @nblumhardt :)

**Pull request merged**